lib/utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+exports.values = function (obj, keys) {
```

Comment:
there are unit tests [here](https://github.com/dwyl/abase/blob/281c8832d2497a491eaf04e2c4fdfb7ed6091fec/test/utils.test.js) for everyone except `except`!  @eliascodes i'll write it?